lib/node/events.ts

```diff
@@ -9,9 +9,9 @@ const EE_LISTENERS = 'listeners';
 const EE_ON = 'on';
 
 
-const zoneAwareAddListener = makeZoneAwareAddListener(EE_ADD_LISTENER, EE_REMOVE_LISTENER, false, true);
-const zoneAwarePrependListener = makeZoneAwareAddListener(EE_PREPEND_LISTENER, EE_REMOVE_LISTENER, false, true);
-const zoneAwareRemoveListener = makeZoneAwareRemoveListener(EE_REMOVE_LISTENER, false);
+const zoneAwareAddListener = makeZoneAwareAddListener(EE_ADD_LISTENER, EE_REMOVE_LISTENER, false, true, true);
```

Comment:
IMO adding a "magic" boolean param is not the best way to to this.

Can you add `makeChainable...` that wraps the underlying method and return self ?